stock_available_to_promise_release_carrier_alternative/models/stock_picking.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import _, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    def _carrier_can_be_changed(self):
```

Comment:
In my opinion, this brings no value. The picking contains moves that are released, so it will always return True

Reply:
> The picking contains moves that are released

Not until they are. Before release, moves can lack availability, and can't be released.
I did that according to your specs.

Reply:
fine, to keep it